idea/meta-storage/src/services/meta/input/create-meta.input.ts

```diff
@@ -0,0 +1,4 @@
+export interface CreateMetaInput {
```

Comment:
I don't see the point of creating and using this interface

Reply:
https://github.com/gear-tech/gear-js/pull/1282#discussion_r1183931979